k2/csrc/tensor.h

```diff
@@ -88,7 +89,7 @@ class Shape {
   // compute the number of elements
   int32_t ComputeNumElement() const;
   int32_t ComputeStorageSize() const;
-  bool CheckContiguous() const;
+  bool RefreshContiguous() const;
```

Comment:
Please dont add functions like this without documenttion!
IsContiguous() is obvious enough, but this function is unclear.

Reply:
I would like the following form:
```cpp
void RefreshContiguous() {
  //  update is_contiguous_;
}

Reply:
Please clarify exactly when it should be called.  Is it to be called only by member functions?
Does the user have an obligation to call it under certain circumstances?

Reply:
These functions are all private.

Reply:
I remember that you commented on this before and you felt that `Check` is not a good function name

Reply:
Replaced with `ComputeIsContiguous`.